src/JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -155,8 +154,7 @@ private async Task<bool> CallMessageHandler(Message message)
             watch.Stop();
 
             _logger.LogTrace(
-                "Handled message with Id '{MessageId}' of type {MessageType} in {TimeToHandle}.",
-                message.Id,
```

Comment:
So here it is, the one place where we actually do something with a property on `Message` (ok, there was `UniqueKey()` too I guess). It's on a trace log, we arguably would want to use the SQS message id for this level of debugging, but tying it back to the domain event is going to be the challenge.